queuebot/cogs/queue/suggestion.py

```diff
@@ -301,9 +304,10 @@ def status(self):
         """
         upvotes = self.record['upvotes']
         downvotes = self.record['downvotes']
-
-        # This logic is copied from b1nb0t.
-        if upvotes >= 10 and upvotes - downvotes >= 5 and upvotes + downvotes >= 15:
+        if upvotes + downvotes <= required_votes:
```

Comment:
Since this is supposed to be the opposite of `upvotes + downvotes >= required_votes`, it needs to be `upvotes + downvotes < required_votes`, not `<=`.

Reply:
woops